lib/git/pr/release/util.rb

```diff
@@ -94,12 +94,13 @@ def build_pr_title_and_body(release_pr, merged_prs, changed_files, template_path
                        DEFAULT_PR_TEMPLATE
                      end
 
-          erb = if RUBY_VERSION >= '2.6'
-                  ERB.new template, trim_mode: '-'
-                else
-                  ERB.new template, nil, '-'
-                end
-          content = erb.result binding
+          erb = ERB.new template, trim_mode: '-'
```

Comment:
IMO: This hack of checking Ruby version is commonly used in other gems.

e.g.
- https://github.com/rubocop/rubocop/blob/v1.12.0/lib/rubocop/cop/lint/erb_new_arguments.rb#L45-L60
- https://github.com/rails/rails/blob/v6.1.3/railties/lib/rails/generators/migration.rb#L66-L70
- https://github.com/rtomayko/tilt/blob/v2.0.10/lib/tilt/erb.rb#L24-L28